hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/utils/ConversionUtils.java

```diff
@@ -173,6 +173,16 @@ public static com.esaulpaugh.headlong.abi.Address headlongAddressOf(@NonNull fin
         return asHeadlongAddress(asEvmAddress(tokenId.tokenNum()));
     }
 
+    /**
+     * Given an entity number, returns it as a headlong address.
```

Comment:
Is this version of the overloaded method ever used/called?

Reply:
I think I used it at one time, not used now.   Removed it.